src/librustc_mir/borrow_check/nll/type_check/constraint_conversion.rs

```diff
@@ -186,6 +169,23 @@ impl<'a, 'gcx, 'tcx> ConstraintConversion<'a, 'gcx, 'tcx> {
             sub,
             sup,
         });
+
+        // In the new analysis, all outlives relations etc
+        // "take effect" at the mid point of the statement
+        // that requires them, so ignore the `at_location`.
+        if let Some(all_facts) = &mut self.all_facts {
```

Comment:
Maybe it would be better if we just iterated over the `outlives_constraints` that we build up at the end? I forget though if there are outlives constraints we don't need for polonius.